### ISSUE TITLE
how to shoot yourself in the foot trying to access Keyboard / Trackpad

### ISSUE THREAD

**Issue by telephon** [ModalityTeam/Modality-toolkit#207]
The following will freeze keyboard-arrow-keys and trackpad access on the whole system. I had to cold-restart the system with no ability to save any files, so this is kind of serious (in my case, I was lucky).

````

// Available HIDMKtlDevices:
// MKtl(autoName);  // [ hid product, vendor(, serial number) ]
    MKtl('ppln0');  // [ "Apple Internal Keyboard / Trackpad", "Apple Inc." ]
    MKtl('ppln1');  // [ "Apple Internal Keyboard / Trackpad", "Apple Inc." ]
    MKtl('ppln3');  // [ "Apple Internal Keyboard / Trackpad", "Apple Inc." ]
    MKtl('ppln2');  // [ "Apple Internal Keyboard / Trackpad", "Apple Inc." ]

// So this is how to do it (shoot yourself in the foot):
 // well, better DON'T DO IT unless you have everything saved!
// k =  MKtl('ppln0');
// k.gui;

````

The GUI (tiny window) shows a little 'nil' on it.

**Comment by adcxyz** [comment]
wow. we should block that from code then.

**Comment by LFSaw** [comment]
HID for internal devices on OSX is completely broken... nothing we can do about apart from blocking them :/

**Comment by adcxyz** [comment]
closed with 63a42ae62f21281f95609fab001aa47e8393935a
